modeller/chapter.md

```diff
@@ -155,13 +161,14 @@ To draw an item to the screen, we need to convert between a few different coordi
 ![Transformation Pipeline](newtranspipe.png?raw=true)
 Thank you very much to Dr. Anton Gerdelan for the image. His OpenGL tutorial book is available [here](http://antongerdelan.net/opengl/).
 
-We let specialized OpenGL functions handle the right hand side of the diagram: conversion from eye space to homogeneous clip space is handled by `gluPerspective`, conversion to Normalized Device Space and Viewport space is handled by `glViewport`. These matrices do not change during program execution.
-Points in the view space are converted into points in 2d using the projection matrix.
+We let specialized OpenGL functions handle the right hand side of the diagram: conversion from eye space to homogeneous clip space is handled by `gluPerspective`, and conversion to normalized device space and viewport space is handled by `glViewport`. These matrices do not change during program execution.
+Points in the view space are converted into points in 2D using the projection matrix.
```

Comment:
Here you refer to "view space" but there is no "view space" in the diagram. You also refer to "model space" on line 168. Can you change the names of space references to match the names used in the diagram? Alternately, you could provide a mapping between your terminology and the diagram's terminology in the explanatory text I suggested above.